### ISSUE TITLE
Select Page Type Dialog Ordering

### ISSUE THREAD

**Issue by dotnetshadow** [KalikoCMS/KalikoCMS.Core#60]
Hi what is the ordering of the Create New Page dialog for page types, I thought it was alphabetical but it looks like it based on date time or PageTypeId?

**Comment by fschultz** [comment]
It should be in alphabetical order but left-right-top-down:
A B
C D
E ..

If not it's a bug :)

**Comment by dotnetshadow** [comment]
![capture](https://cloud.githubusercontent.com/assets/296247/9952037/0c7eeb4a-5e13-11e5-8892-4da12571807c.JPG)

I think there could be a bug. In the above screenshot Article Page should come before Article Page - Landing. I also tryied to rename Article Page - Landing to XYZ including the class name, name etc but it didn't change.

I checked your source for that dialog and it didn't like there was a sort function applied to it?
